### PR TITLE
Upgrade to v3.2.0 of the AWS Terraform Provider

### PR DESCRIPTION
**Pull request by stack72**
Fixes: #1074

### CODE REVIEW

File: examples/examples_nodejs_test.go [hunk 1]
```diff
@@ -173,9 +173,8 @@ func TestAccAlbNew(t *testing.T) {
 			// RunUpdateTest: true,
 			EditDirs: []integration.EditDir{
 				{
-					Dir:             "step2",
-					Additive:        true,
-					ExpectNoChanges: true,
```
Comment (leezen) [inline examples/examples_nodejs_test.go:178]: Why does this change?

File: examples/examples_py_test.go [hunk 2]
```diff
@@ -44,9 +44,8 @@ func TestAccAlbLegacyPy(t *testing.T) {
 					RunUpdateTest: true,
 					EditDirs: []integration.EditDir{
 						{
-							Dir:             "step2",
-							Additive:        true,
-							ExpectNoChanges: true,
```
Comment (leezen) [inline examples/examples_py_test.go:49]: Why does this change?

File: examples/examples_py_test.go [hunk 3]
```diff
@@ -65,9 +64,8 @@ func TestAccAlbNewPy(t *testing.T) {
...
```
Comment (leezen) [inline examples/examples_py_test.go:70]: Why does this change?

File: sdk/nodejs/types/output.ts [hunk 4]
```diff
@@ -6488,7 +6542,7 @@ export namespace ec2 {
 
     export interface GetLaunchTemplateNetworkInterface {
         associatePublicIpAddress?: boolean;
-        deleteOnTermination: boolean;
+        deleteOnTermination: string;
```
Comment (leezen) [inline sdk/nodejs/types/output.ts:6545]: Is this change expected?

File: sdk/go/aws/ec2/pulumiTypes.go [hunk 5]
```diff
@@ -14632,8 +14689,8 @@ func (o GetLaunchTemplateMonitoringArrayOutput) Index(i pulumi.IntInput) GetLaun
 }
 
 type GetLaunchTemplateNetworkInterface struct {
-	AssociatePublicIpAddress *bool `pulumi:"associatePublicIpAddress"`
-	DeleteOnTermination      bool  `pulumi:"deleteOnTermination"`
+	AssociatePublicIpAddress *bool  `pulumi:"associatePublicIpAddress"`
+	DeleteOnTermination      string `pulumi:"deleteOnTermination"`
```
Comment (leezen) [inline sdk/go/aws/ec2/pulumiTypes.go:14693]: (Also left a comment on the TS side)

File: examples/examples_nodejs_test.go (hunk 1 above)
Comment (stack72) [reply examples/examples_nodejs_test.go:178]: Subnets are now a computed field so changes outside of the RouteTable will cause diffs

File: examples/examples_py_test.go (hunk 2 above)
Comment (stack72) [reply examples/examples_py_test.go:49]: Subnets are now a computed field so changes outside of the RouteTable will cause diffs

File: examples/examples_py_test.go (hunk 3 above)
Comment (stack72) [reply examples/examples_py_test.go:70]: Subnets are now a computed field so changes outside of the RouteTable will cause diffs

File: sdk/nodejs/types/output.ts (hunk 4 above)
Comment (stack72) [reply sdk/nodejs/types/output.ts:6545]: This is not - we need to override this as its due to their "nullable bool" approach

File: sdk/go/aws/ec2/pulumiTypes.go (hunk 5 above)
Comment (stack72) [reply sdk/go/aws/ec2/pulumiTypes.go:14693]: This is not - we need to override this as its due to their "nullable bool" approach

**Review by leezen — APPROVED** [review]

**Pull request merged**